src/en/charms-scaling.md

```diff
@@ -163,49 +139,46 @@ assigned an ID of '3'.
 Read the [Using constraints][charms-constraints] page for details on
 constraints.
 
-## Scaling Back
+## Scaling down
 
-Sometimes you may want to scale back some of your applications, and this too is
-easy with Juju.
+In the context of Juju, scaling down means decreasing the number of application
+units and always involves the `remove-unit` command.
 
-The general usage to scale down an application is with the `remove-unit` command:
+Closely resembling scaling down is the direct removal of a machine. This is
+therefore also covered here and is accomplished via the `remove-machine`
+command.
+  
+To scale down the MediaWiki application by removing a specific unit:
 
 ```bash
-juju remove-unit [options] <unit> [...]
+juju remove-unit mediawiki/1
 ```
 
-For example, the following scales down the MediaWiki application by removing a
-specific unit:
+Note that because this is the only unit running on the underlying machine, the
+machine will also be removed.
 
-```bash
-juju remove-unit mediawiki/1
-```
+A machine can be manually removed unless any of the following is true for that
```

Comment:
I find it difficult to follow these positive and negative assertions, *can be manually removed*, *unless any of the following*, *is true*. As above, could this be untwisted?